lsm-tree.cabal

```diff
@@ -34,7 +34,14 @@ common wno-x-partial
 common language
   default-language:   GHC2021
   default-extensions:
+    DeriveAnyClass
```

Comment:
This one conflicts with `GenericNewtypeDeriving`. See the build failures.

Reply:
Arguably we could enable `-Wmissing-deriving-strategies` (with `Werror`).